src/main/java/com/solidbrain/russads/AutomaticTaskCreator.java

```diff
@@ -0,0 +1,131 @@
+package com.solidbrain.russads;
+
+import static com.solidbrain.russads.configuration.ConfigurationHelper.AUTO_SALES_REP_KEY;
+
+import java.text.DateFormat;

```

Comment:
We strive to use new Java Date/Time API (http://www.threeten.org/) instead of old one.